documentation/release_notes.markdown

```diff
@@ -10,6 +10,18 @@ canonical: "/puppetserver/latest/release_notes.html"
 [puppetserver.conf]: ./config_file_puppetserver.markdown
 [product.conf]: ./config_file_product.markdown
 
+## Puppet Server 6.12.0
+
+Released 3 June 2020
+
+### Resolved Issue
+
+- JRuby has been bumped to 9.2.11.1 again, with `invokedynamic.yield` set to false to resolve a stackoverflow error. [SERVER-2793](https://tickets.puppetlabs.com/browse/SERVER-2793)
+
+### Deprecation
+
+- We have deprecated the v1 metrics endpoint, which was recently disabled by default. It will be removed entirely in a future release. Please use the v2 endpoint instead. [TK-486](https://tickets.puppetlabs.com/browse/TK-486)
```

Comment:
```suggestion
- The v1 metrics endpoint, which was recently disabled by default, is now deprecated. Instead, use the v2 endpoint. [TK-486](https://tickets.puppetlabs.com/browse/TK-486)
```

Reply:
Usually we avoid saying things about future releases in the docs, and being deprecated implies that it will be removed later anyway I think.

Reply:
Good point, thanks! updated 😊